config/locales/views/en.yml

```diff
@@ -462,6 +498,22 @@ en:
           comments_on_works: delete or hide comments they previously left on your works; you can delete these individually
           hide_works: hide their works or bookmarks from you
           intro: 'Blocking a user will not:'
+  challenge:
+    gift_exchange:
+      challenge_signups:
+        close_offers_html: Close Offers &#8593;
+        close_requests_html: Close Requests &#8593;
+        heading:
+          for_collection: Sign-ups for collection %{collection}
+          for_search: Sign-up
```

Comment:
This is actually a countable thing -- it appears as "1 Sign-up" (or however many results are returned) because it's part of search header. For example: https://test.archiveofourown.org/collections/a_quick_test/signups?query=testy&commit=Search+By+Pseud

Reply:
That will get pluralised by https://github.com/otwcode/otwarchive/blob/18dd6501b29e3a107ade83cf15f13aa58c83147b/app/helpers/search_helper.rb#L9